apps/src/code-studio/videos.js

```diff
@@ -400,15 +392,9 @@ videos.onYouTubeBlocked = function (youTubeBlockedCallback, videoInfo) {
   testImageAccess(
     youTubeAvailabilityEndpointURL(noCookie) + '?' + Math.random(),
     // Called when YouTube availability check succeeds.
-    function () {
-      // Track event in Google Analytics.
-      trackEvent('showvideo', 'startVideoYouTube', key);
-    },
-
+    function () {},
```

Comment:
Probably unneeded, but just asking - would it be worthwhile to name these functions?

w/o being sure of exactly how this is used, I know that you can get reports in logs that just say "anonymous function", whereas if it was named it might be a more useful report "testImageAccessTracker" or something.

Or those may not show up in this case making it moot. 😄

Reply:
Good point.  GA4 doesn't report function names, so this is probably alright.